oncokb_annotator_3.2.2/oncokb_annotator_3-2-2.cwl

```diff
@@ -0,0 +1,90 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: oncokb_annotator
+baseCommand:
+  - python3
+  - /oncokb/MafAnnotator.py
+inputs:
+  - id: inputMafFile
+    type: File
+    inputBinding:
+      position: 0
+      prefix: '-i'
+    doc: input maf file for annotation
+  - id: outputMafName
+    type: string
+    inputBinding:
+      position: 0
+      prefix: '-o'
+  - id: apiToken
```

Comment:
Lets make the API toke as a file that you read on the fly within CWL to make a string